### DISCUSSION TITLE
convert command does not work after installation

### DISCUSSION THREAD

**Discussion by LinusHojaGER** [ImageMagick/ImageMagick#3722] [Help] [answered]
I built and installed ImageMagick according to  https://github.com/ImageMagick/ImageMagick/blob/main/Install-mac.txt 
I need to build it from source, since i do need a specific build.
I had no errors, everything works fine.
But I cant execute "convert" after the installation. It always fails with "bash: convert: command not found".
Even after a restart of the console it does not work.

I am working on MacOS Mojave 10.14.6

Update:
I just checked my file-system and in /opt i have the following ImageMagick-related files:
<img width="1318" alt="Bildschirmfoto 2021-05-22 um 21 10 10" src="https://user-images.githubusercontent.com/58180468/119238360-57c9a300-bb42-11eb-885f-63b35e49d2e6.png">
So it should normally work, right? Am I missing something?

**Comment by fmw42** [comment]
Which method did you use?

**Reply by LinusHojaGER** [reply]
Method 2: Compile from source

**Reply by fmw42** [reply]
If you compile from source, then you have to install all the delegate libraries (such as for PNG, TIFF, JPG, etc) from source, before you install ImageMagick. So I would not suggest that you try to do that. You will need to install from Homebrew or from ImageMagick binary.  Alternately, install all your delegates using MacPorts and then install ImageMagick from source. The latter is what I do, but using Homebrew would be much easier.

**Reply by LinusHojaGER** [reply]
Yes I know, this is not the problem, i can build and install it without errors, just the convert command does not work. I really need that to work, because its for my final project of my apprenticeship, and there is no other way to be able to install a exact hotfix version on Mac. With HomeBrew I am only able to install the latest minor version of a major version. eg 6 or 7 and i need to be able to specify the minor and hofix version: 7.0.1 for example.

I always get "bash: convert: command not found".

Is it maybe important where I perform the installation? At the moment I am just doing it at any Location.

**Reply by fmw42** [reply]
There must be some error. Check the config.log and see if that shows any error messages.

**Reply by LinusHojaGER** [reply]
No, exit code 0. Also I can't find any errors.

**Comment by snibgo** [comment]
Did you use "./configure"? If so, did you use "--enable-legacy-support"?

**Reply by LinusHojaGER** [reply]
I used "sudo ./configure --prefix=/opt --with-quantum-depth=16 --disable-dependency-tracking --without-perl".
After your comment I also added "--enable-legacy-support", but it still didn't work.

**Reply by LinusHojaGER** [reply]
Thanks for your help, resolved it.

**Comment by fmw42** [comment]
What was your exact convert command?

**Reply by LinusHojaGER** [reply]
convert --version

**Reply by LinusHojaGER** [reply]
Thanks for your help. resolved it.

**Comment by LinusHojaGER** [comment, marked as answer]
Just got it to work. I had to remove "--prefix=/opt" when executing ./configure
